x-pack/elastic-agent/pkg/agent/application/fleet_server_bootstrap.go

```diff
@@ -133,6 +136,7 @@ func (b *FleetServerBootstrap) Start() error {
 	if err := b.srv.Start(); err != nil {
 		return err
 	}
+	// TODO: Does it make sense to pass in a ctx here?
```

Comment:
Can you either clarify the TODOs and remove them or add a bit more detail, your github handle and/or a link to a follow up task?

Reply:
:+1: i should start writing "QUESTION" since that's what they really are, they just get highlighed when I use `todo` in vim so i see them :sweat_smile: i've cleaned it up a bit since talking with @michel-laterman last week.